e2e/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Use the official Playwright image as a base
+FROM mcr.microsoft.com/playwright:v1.48.1-noble
+
+# Install make and any other necessary dependencies
+RUN apt-get update && apt-get install -y make
+
+# Set the working directory to the root of the project inside the Docker container
+WORKDIR /app
+
+# Copy the required files and directories from the root of the project into the Docker container
+COPY Makefile /app/Makefile
+COPY e2e /app/e2e
+
+# Install the dependencies inside the /e2e directory
+WORKDIR /app/e2e
+RUN npm ci
+
+# Install Playwright browsers and other necessary system dependencies
+RUN npx playwright install --with-deps
+
+# Change the working directory back to the root to run the Makefile
+WORKDIR /app
```

Comment:
nit: the /app folder is a little confusing since it doesn't map to the concept of applications in the rest of the project, so to simplify let's just put the Makefile in the root folder and the e2e folder in /e2e e.g.

```suggestion
WORKDIR /

# Copy the required files and directories from the root of the project into the Docker container
COPY Makefile /Makefile
COPY e2e /e2e

# Install the dependencies inside the /e2e directory
WORKDIR /e2e
RUN npm ci

# Install Playwright browsers and other necessary system dependencies
RUN npx playwright install --with-deps

# Change the working directory back to the root to run the Makefile
WORKDIR /
```